cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casInstitutionLoginView.jsp

```diff
@@ -87,6 +87,8 @@
                 institutionLoginUrl = "${callutheranUrl}";
             } else if (institutionLoginUrl === "okstate") {
                 institutionLoginUrl = "${okstateUrl}";
+            } else if (institutionLoginUrl === "cord") {
+                institutionLoginUrl = "${cordUrl}"
```

Comment:
* Step3: As shown in the institution model [OSF PR LINK TBD](), institution SSO using the CAS / `pac4j` do not have a pre-configured login URL stored in the DB. It is initiated and calculated by CAS. Thus, must set the login URL when users select an institution and click the sign-in button.